docs/cloud_providers/aws/index.md

```diff
@@ -0,0 +1,76 @@
+# cloud-provider-aws
```

Comment:
Using index.md as the file name is not aligned with other docs. The file name could be for example cloud-provider-aws.md to follow similar naming (or then do a bigger refactor).